application-analytics-ui/src/main/resources/Analytics/Code/Configuration.xml

```diff
@@ -61,12 +61,26 @@
         <picker>1</picker>
         <prettyName>Authentication Token </prettyName>
         <size>30</size>
-        <storageType>Hash</storageType>
+        <storageType>Clear</storageType>
```

Comment:
Why do you need these changes (+ the properties rearranged) ? Maybe you recreated the document and these aren't in the same order now